xbmc/windowing/X11/GLContextGLX.cpp

```diff
@@ -153,10 +153,6 @@ bool CGLContextGLX::IsSuitableVisual(XVisualInfo *vInfo)
     return false;
   if (glXGetConfig(m_dpy, vInfo, GLX_BLUE_SIZE, &value) || value < 8)
     return false;
-  if (glXGetConfig(m_dpy, vInfo, GLX_ALPHA_SIZE, &value) || value < 8)
-    return false;
-  if (glXGetConfig(m_dpy, vInfo, GLX_DEPTH_SIZE, &value) || value < 8)
```

Comment:
why do you drop depth check?

Reply:
Was done by mistake. 8 is enough? For EGL we set it to 16 on certain platforms and to 24 on others. What do we really need?

Reply:
it should be 24, aligned with this: https://github.com/xbmc/xbmc/blob/master/xbmc/windowing/X11/GLContextEGL.cpp#L284

Reply:
Thx. made the check for 24